tests/app/validation/test_month_year_validator.py

```diff
@@ -83,7 +83,7 @@ def test_month_year_date_validator_invalid_year(self):
 
         mock_form = Mock()
         mock_form.month.data = "12"
-        mock_form.year.data = "17"
```

Comment:
Does the fact 17 no longer fails this test mean we need to consider validation around years with only 2 digits.